configs/components/openssl-1.1.1.rb

```diff
@@ -45,7 +45,7 @@
     pkg.environment 'CC', '/opt/pl-build-tools/bin/gcc'
 
     cflags = '$${CFLAGS} -static-libgcc'
-    ldflags = "#{settings[:ldflags]} -WL, -R#{settings[:libdir]}"
+    ldflags = "#{settings[:ldflags]} -Wl, -R#{settings[:libdir]}"
```

Comment:
Were you able to build the runtime with this change? The reason I ask is I'm not used to seeing spaces between `-Wl` and the rest:

```suggestion
    ldflags = "#{settings[:ldflags]} -Wl,-R#{settings[:libdir]}"
```

The idea is `-Wl` tells gcc to pass the arguments that *directly* follow to the linker. But if there's a space after the comma it may not work as expected?

Reply:
Yeah the space doesn't work. I'll force push to your branch, but next time can you try to build before marking it as "Ready for review"?

```
Makefile:111: recipe for target 'openssl-1.1.1-build' failed
gcc: error: unrecognized command line option '-R'
gmake[2]: *** [apps/app_rand.o] Error 1
gmake[1]: *** [all] Error 2
gmake: *** [openssl-1.1.1-build] Error 2
```

Reply:
I am new to using  `gcc` command options. For other commands I used spaces between the  options so took it casually. thanks for catching this Josh! B-)